report/src/main/webapp/app/layout/toolbar/toolbarBottom.view.html

```diff
@@ -57,18 +57,28 @@ <h4 class="toolbar-link-header ellipsis">
                 <i class="glyphicon glyphicon-comment"></i>
             </span>
 
+            <span class="button button-dark button-disabled"
+                  data-ng-if="toolbarBottom.acceptButtonDisabled"
+                  data-aet-included-comment-popover
+                  data-toggle="popover"
+                  data-content="May not be accepted because compared against another suite patterns."
+                  data-html="true"
+                  data-trigger="hover"></i>
+                Accept {{toolbarBottom.viewMode}}
+            </span>
+
             <span class="button js-rebase-suite button-blue"
-                  data-ng-if="toolbarBottom.showAcceptButton()"
+                  data-ng-if="toolbarBottom.showAcceptButton() && !toolbarBottom.acceptButtonDisabled"
```

Comment:
Maybe we should put `!toolbarBottom.acceptButtonDisabled` logic inside `toolbarBottom.showAcceptButton()`?
And the same below for `toolbarBottom.showRevertButton()`.